kimchi/src/proof.rs

```diff
@@ -249,6 +254,9 @@ impl<F: FftField> ProofEvaluations<Vec<F>> {
     pub fn combine(&self, pt: F) -> ProofEvaluations<F> {
         ProofEvaluations::<F> {
             s: array::from_fn(|i| DensePolynomial::eval_polynomial(&self.s[i], pt)),
+            coefficients: array::from_fn(|i| {
```

Comment:
I thought `coefficients` were already an evaluation of the 15 coefficient polynomials at a challenge point. Thus, I am not sure I understand why `combine` uses those field elements to compute `c0·pt^0 + c1·pt + c2·pt^2 + ... + c14·pt^14`. Then the question is more like: what do these 15 field elements represent?

Reply:
(this is for when we split polynomials if I remember correctly, which we don't use atm)

check "coefficients" under the spec here: https://o1-labs.github.io/proof-systems/specs/kimchi.html#tables-used-to-describe-a-circuit